### PR TITLE
[19.1] Removed elements

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

- Deprecated grid
- Action icons
- Deprecated empty states
- Button's mod-icon
- u-comma
- u-unit
- Box shadow / deprecated elevations
- Tshirt spacings

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3302/storybook

File: packages/scss/src/commons/core.scss [hunk 1]
```diff
@@ -34,40 +34,31 @@ $overflow: 'hidden', 'auto', 'visible', 'scroll';
...
 	}
 }
 
-@mixin spacings($token: true) {
-	$spacings: config.$spacingsDeprecated;
-	@if $token {
-		$spacings: config.$spacings;
-	}
+@mixin spacings() {
+	$spacings: config.$spacings;
```
Comment (vvalentin-lucca) [inline packages/scss/src/commons/core.scss:44]: You don't need to go through an intermediate variable, you can directly use `config.$spacing`.

File: README.md [hunk 2]
```diff
@@ -32,7 +32,6 @@ In your file styles.scss, add imports files and components you want to import to
 @forward '@lucca-front/ng/src/main';
 
 // Import SCSS components
-@forward '@lucca-front/scss/src/components/actionIcon';
```
Comment (vvalentin-lucca) [inline README.md:35]: Why is this line being removed?

**Review by vvalentin-lucca — APPROVED** [review]

**Review by shajz — APPROVED** [review]
🔥

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-3302/storybook

**Pull request merged**